pandas/tests/resample/test_period_index.py

```diff
@@ -877,3 +891,30 @@ def test_sum_min_count(self):
             [3.0, np.nan], index=PeriodIndex(["2018Q1", "2018Q2"], freq="Q-DEC")
         )
         tm.assert_series_equal(result, expected)
+
+
+def test_resample_frequency_ME_error_message(series_and_frame):
+    msg = "Invalid frequency: 2ME"
+
+    obj = series_and_frame
+    with pytest.raises(ValueError, match=msg):
+        obj.resample("2ME")
+
+
+def test_resample_from_offset_deprecated():
+    msg = "Invalid frequency: 2ME"
+
+    s = Series(range(10), index=date_range("20130101", freq="d", periods=10))
+    with pytest.raises(ValueError, match=msg):
+        s.to_period().resample("2ME")
```

Comment:
these two tests look the same, as you're calling `to_period` in the second one - I think just the one should be enough?